### PR TITLE
socat: new package at 1.7.3.3

### PR DESCRIPTION
**Pull request by aaronNGi**
## Description of package

socat - Multipurpose relay (SOcket CAT)

## Checklist

- [x] Latest upstream version.
- [ ] I agree to maintain this package (**required**).
- [ ] I agree to notify KISS if I can no longer maintain this package (**required**).

### CODE REVIEW

File: community/socat/sources [hunk 1]
```diff
@@ -0,0 +1 @@
+http://www.dest-unreach.org/socat/download/socat-1.7.3.3.tar.gz
```
Comment (dylanaraps) [inline community/socat/sources]: No `https` source available?

Comment (aaronNGi) [reply community/socat/sources]: Apparently they use a self-signed cert. We could also use https://repo.or.cz/socat.git/snapshot/e1bc39a53f808f4fdaeb3743c415da861789f351.tar.gz which is the snapshot of 1.7.3.3. What do you think?

Comment (dylanaraps) [reply community/socat/sources]: `http` is fine if no `https` is available as we also have the checksum verification to fallback on. :+1:

File: community/socat/files [hunk 2]
```diff
@@ -0,0 +1 @@
+socat-1.7.3.3.tar.gz
```
Comment (dylanaraps) [inline community/socat/files]: This file can be removed entirely.

Comment (aaronNGi) [reply community/socat/files]: Oops, brainfart. Fixed.

**Comment by dylanaraps** [comment]
Thanks :+1:

**Pull request merged**